### PR TITLE
Fix UIs, Re-added Brush and Flood, added Cylinder

### PR DESCRIPTION
**Pull request by inxomnyaa**
- MagicWE2 is now using customui to show Brush and Flood forms
- Added //cyl and cylinder brush shape
- Fixed a translation mistake in the default shape
- Fixed NBT -> Array for Item Lore (Brush & Flood item info)
- Register Brush and Flood command only if DEVirion is installed (this will only work for devs!)
- Optimize flood tool corner & block detection algorythm to take the actual limit into accountance and not get stuck anymore. Also, all blocks in encapsulated areas fill now. (If there are any leaks in the "border", a circle-alike shape will generate. On mistakes //undo will work.)

### CODE REVIEW

File: plugin.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 name: MagicWE2
 main: xenialdan\MagicWE2\Loader
-version: 6.0.2
-api: [4.0.0+dev]
+version: 7.0.0
+api: [4.0.0,3.0.0]
```
Comment (Aericio) [inline plugin.yml]: As @SalmonDE stated, changes in 4.0.0 are not backwards compatible with 3.0.0.

i.e. `https://github.com/pmmp/PocketMine-MP/blob/master/src/pocketmine/lang/Language.php` > `https://github.com/pmmp/PocketMine-MP/blob/mcpe-1.8/src/pocketmine/lang/BaseLang.php`

File: src/xenialdan/MagicWE2/Loader.php [hunk 2]
```diff
@@ -5,10 +5,15 @@
 namespace xenialdan\MagicWE2;
 
 use pocketmine\lang\Language;
```
Comment (Aericio) [inline src/xenialdan/MagicWE2/Loader.php:7]: except your still using 3.0.0 path, lol

File: plugin.yml (hunk 1 above)
Comment (inxomnyaa) [reply plugin.yml]: Sadly i must have it enabled because my 1.8 pmmp source uses the 3.0.0 api

File: src/xenialdan/MagicWE2/Loader.php (hunk 2 above)
Comment (inxomnyaa) [reply src/xenialdan/MagicWE2/Loader.php:7]: In the src code of the mcpe-1.8 branch that i use this is already Language

File: plugin.yml (hunk 1 above)
Comment (Aericio) [reply plugin.yml]: rip

**Comment by SuperAdam47** [comment]
finish paste schematic command

**Comment by inxomnyaa** [comment]
@superadam47 there is a schem load command. You paste with //paste.

**Comment by inxomnyaa** [comment]
@aericio Look, i dropped 4.0.0 support :P with that all these issues should not to be worried about anymore, right?

**Pull request merged**